WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/ShippingLabelFormViewModel.swift

```diff
@@ -43,12 +43,14 @@ final class ShippingLabelFormViewModel {
             return nil
         }
 
+        let weight = Double(truncating: (Decimal(string: totalPackageWeight ?? "0") as  NSNumber?) ?? 0)
```

Comment:
Not sure if I understanding this completely, but I think you can get way with a simpler API like:

```suggestion
        let weight = Double(totalPackageWeight ?? "0") ?? .zero
```